### PR TITLE
Docs: Making fixes in README

### PR DESCRIPTION
**Pull request by kelbrown20**
**Description:** Cleaning up some sections and titles in the README for readability. 

Additional info: Fixing the fails

### CODE REVIEW

**Review by nathan-weinberg — APPROVED** [review]
LGTM, adding @russellb for review as well since he made several of these recent changes

File: README.md [hunk 1]
```diff
@@ -13,30 +13,23 @@
...
-    - [Example output](#example-output-3)
-  - [👩‍🏫 Train the model](#-train-the-model)
+  - [👩‍🏫 Training the model](#-training-the-model)
     - [Train the model locally on Linux](#train-the-model-locally-on-linux)
-    - [Train the model locally on an M-series Mac](#train-the-model-locally-on-an-m-series-mac)
-    - [Training the model locally with GPU acceleration](#training-the-model-locally-with-gpu-acceleration)
-    - [Training the model in the cloud](#training-the-model-in-the-cloud)
+    - [Train the model locally on M-series Macs](#train-the-model-locally-on-an-m-series-mac)
+    - [Train the model locally with GPU acceleration](#training-the-model-locally-with-gpu-acceleration)
+    - [Train the model in the cloud](#training-the-model-in-the-cloud)
```
Comment (russellb) [inline README.md:32]: This table-of-contents is automatically generated and updated by a vscode plugin. (Or it is for me!)

This last line at least appears to change the ToC but not the actual heading, so it's going to get changed back when someone else edits the file.

**Review by russellb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kelbrown20) [reply README.md:32]: Hmm the toc does not automatically update for me like that.
I do see theres a inconstancy in that though so I can fix that!

Comment (kelbrown20) [reply README.md]: @russellb just updated!

Comment (russellb) [reply README.md]: @kelbrown20 if you use vscode, try this: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

**Review by russellb — APPROVED** [review]

**Pull request merged**